homeassistant/components/notify/html5.py

```diff
@@ -131,29 +140,79 @@ def post(self, request):
         return self.json_message("Push notification subscriber registered.")
 
 
+class HTML5PushCallbackView(HomeAssistantView):
```

Comment:
Oh i'd also add that actions can be defined dynamically in HTML5 but must be defined ahead of time for iOS.